napari/_qt/event_loop.py

```diff
@@ -9,6 +9,14 @@
 from ..utils.perf import perf_config
 from .exceptions import ExceptionHandler
 
+APPLICATION_NAME = 'napari'
+try:
+    from magicgui.application import APPLICATION_NAME as MGUI_APP_NAME
+except (ImportError, AttributeError):
```

Comment:
Do you know how long this import takes? We put a lot of work during #758 to reducing imports/ making lazy to improve startup time, and it looks like this would run everytime we launch even if not using magicgui. (Not sure if we're already importing magicgui here too https://github.com/napari/napari/blob/master/napari/utils/_magicgui.py and so maybe this doesn't change anything, but I thought I should ask).


The alternative I see is just hardcoding it to `magicgui`, and if that ever changes then we need to make an update.

Thoughts @jni @tlambert03?

Reply:
> Do you know how long this import takes? We put a lot of work during #758 to reducing imports/ making lazy to improve startup time, and it looks like this would run everytime we launch even if not using magicgui. 

takes about 15ms on my computer... I'm totally fine with making it a string though.

Reply:
15ms isn't too bad.

Reply:
I'd still vote for string, but final decision can be @tlambert03 and @jni

Reply:
let's go with string.  It's already a bandaid 😂

Reply:
there.  now it's a one-liner